cmd/krew/cmd/internal/setup_check_test.go

```diff
@@ -43,6 +43,31 @@ func TestIsBinDirInPATH_secondRun(t *testing.T) {
 	}
 }
 
+// Test case when PATH content is not well normalized
+func TestIsBinDirInPATH_Custom(t *testing.T) {
+	tempDir := testutil.NewTempDir(t)
+	backupPath := os.Getenv("PATH")
+
+	err := os.Setenv("PATH", tempDir.Root()+"/home/../home/user//./.krew/bin")
+	if err != nil {
+		t.Errorf("fail to set PATH with not clean path")
```

Comment:
```suggestion
		t.Fatalf("fail to set PATH with not clean path")
```
There's no point to continue the test if the setup failed.